### PR TITLE
Ensure PR labels and provide PR template

### PR DESCRIPTION
**Pull request by britsta**
It would be good if we could use labels for pull requests in the documentation for standardization with the other repositories and also for clarity. A default checklist and a placeholder for a description for pull requests is also helpful.

### CODE REVIEW

File: .github/workflows/check-pr.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  validate-pr-label:
+    name: Validate the Pull Request's labels
+    runs-on: ubuntu-latest
+    steps:
+      - name: PR must be labeled
+        uses: jesusvasquez333/verify-pr-label-action@v1.4.0
+        with:
+          disable-reviews: true
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+          valid-labels: "breaking-change, bug, chore, ci, dependencies, documentation, enhancement, refactoring, test"
```
Comment (jkoenig134) [inline .github/workflows/check-pr.yml:17]:
```suggestion
          valid-labels: "bug, chore, ci, dependencies, documentation, enhancement, refactoring"
```

i don't think we have tests or breaking changes?

Comment (britsta) [reply .github/workflows/check-pr.yml:17]: I also thought about it at first and removed them. Then I did some more research and it seems that the labels **could** make sense after all. For example, if we add a migration guide and mix up the documentation, "breaking change" as label could be appropriate.

However, I think that the other labels such as "documentation" and "enhancement" are not so good in coexistence either in the documentation repository. I would be in favor of rethinking from scratch which labels make sense and which don't, and then revising them altogether. The same applies to the other repositories.

Comment (jkoenig134) [reply .github/workflows/check-pr.yml:17]: oh - right, documentation does not make sense at all in this repo as all documentation things here are enhancements!

Comment (jkoenig134) [reply .github/workflows/check-pr.yml:17]: I'd say let's get started with a minimum set of labels we think we need (please sync them to the settings file too) and expand it if we need more.

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -11,4 +11,6 @@ RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/
 #     && apt-get -y install --no-install-recommends <your-package-list-here>
 
 # [Optional] Uncomment this line to install global node packages.
-# RUN su vscode -c "source /usr/local/share/nvm/nvm.sh && npm install -g <your-package-here>" 2>&1
\ No newline at end of file
+# RUN su vscode -c "source /usr/local/share/nvm/nvm.sh && npm install -g <your-package-here>" 2>&1
+
+RUN echo '\nsource /usr/share/bash-completion/completions/git' >> /home/vscode/.bashrc
```
Comment (britsta) [inline .devcontainer/Dockerfile:16]: @sebbi08 helped me to add completions for git in my bash terminal in the documentation repository and adding this line in the Dockerfile was a possible solution. @jkoenig134 is there another solution or is it okay to add this specific line here?

Comment (jkoenig134) [reply .devcontainer/Dockerfile:16]: I don't care, but isn't this the wrong PR to add this?

**Review by jkoenig134 — APPROVED** [review]

File: .devcontainer/Dockerfile (hunk 2 above)
Comment (britsta) [reply .devcontainer/Dockerfile:16]: You got me. :D I thought it would be overkill if we opened a separate PR for this single line.  :)

**Pull request merged**